app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'info@wbooks.com'
```

Comment:
There is no need to change it, but just for your information, this is preferable to be taken from an environment variable :)